Packages/vcs/Lib/configurator.py

```diff
@@ -167,12 +102,18 @@ def __init__(self, canvas):
         self.anim_button = None
         self.listeners = []
         self.animation_last_frame_time = datetime.datetime.now()
+        self.picker = vtk.vtkPropPicker()
         # Map custom templates to their source template
         self.templates = {}
 
         self.creating = False
         self.click_locations = None
 
+    @property
+    def render_window(self):
```

Comment:
Ran into a circumstance where I needed the render window before the interactor was initialized, so I added this property to save some pain and agony getting a hold on the render window.